lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -286,11 +287,13 @@ class ExportCommand < HammerCLIForeman::Command
 
       build_options
 
+      # rubocop:disable Metrics/MethodLength
```

Comment:
We should absolutely break up this method; It's way too long and complicated.

Reply:
Yeah I agree, I will do that on the next push with the test fixes.